sci-misc/boinc/boinc-7.9.3.ebuild

```diff
@@ -0,0 +1,196 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
This is giving avay that you do not use `repoman commit` ;) I highly recommend it as it detects QA stuff like that (in this case even auto-fixes it).